src/Internal.AspNetCore.BuildTools.Tasks/JsonPeek.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNetCore.BuildTools
+{
+#if SDK
+    public class Sdk_JsonPeek : Task
+#elif BuildTools
+    public class JsonPeek : Task
```

Comment:
John told me he wanted something like this for building the .deb and .rpm runtime store installers. I'm gonna hold off on merging till he can take a look to see if this addresses his need.